alchemy-web/src/content/docs/providers/cloudflare/worker.mdx

```diff
@@ -115,516 +210,376 @@ And register `env.d.ts` in your `tsconfig.json`'s `types`.
+**When to use:** Background job processing, webhook handling, email processing, image optimization, data synchronization.
 
-- **`"node"`**: Enables Node.js compatibility by including the `nodejs_compat` flag
+## Smart Placement (performance optimization)
 
-### Combining with Custom Flags
-
-Preset flags are automatically combined with any custom `compatibilityFlags` you provide:
+Enable automatic network optimization by configuring smart placement to reduce latency and improve performance:
 
 ```ts
-import { Worker } from "alchemy/cloudflare";
-
-const worker = await Worker("advanced-worker", {
-  name: "advanced-worker", 
+export const optimizedWorker = await Worker("api", {
   entrypoint: "./src/api.ts",
-  compatibility: "node", // Adds nodejs_compat
-  compatibilityFlags: ["experimental_feature"], // Additional flags
-  // Result: ["nodejs_compat", "experimental_feature"]
+  placement: {
+    mode: "smart"  // Automatically optimize placement for performance
+  }
 });
 ```
 
-## Smart Placement
-
-Enable smart placement to automatically optimize your worker's placement across Cloudflare's global network:
-
-```ts
-import { Worker } from "alchemy/cloudflare";
+**Benefits:**
+- ✅ Automatic network optimization based on performance metrics
+- ✅ Reduced latency for global users
+- ✅ No infrastructure management required
+- ✅ Seamless scaling across Cloudflare's network
 
-const worker = await Worker("optimized-worker", {
-  name: "optimized-worker",
-  entrypoint: "./src/api.ts",
-  placement: {
-    mode: "smart", // Enable smart placement
-  },
-});
-```
+**When to use:** Global applications, latency-sensitive APIs, high-traffic workloads, applications with geographically distributed users.
 
 :::note
-Learn more about [Smart Placement](https://developers.cloudflare.com/workers/configuration/smart-placement/) in the Cloudflare documentation.
+You can learn more about [Smart Placement](https://developers.cloudflare.com/workers/configuration/smart-placement/) in the Cloudflare documentation.
 :::
 
-## CPU Time Limit
+## Local development & configuration
 
-Set the maximum CPU time in milliseconds that the worker can use:
+Mirror production bindings in emulation, schedule tasks with cron, and serve static assets:
 
 ```ts
-
-const worker = await Worker("api", {
-  name: "api-worker",
-  entrypoint: "./src/api.ts",
-  limits: {
-    cpu_ms: 300_000,
-  },
+// Local development configuration
+await Worker("api", { 
+  entrypoint: "./src/api.ts", 
+  dev: { port: 8787 } 
 });
-```
 
-:::note
-Learn more about [CPU Time Limits](https://developers.cloudflare.com/workers/platform/limits/#cpu-time) in the Cloudflare documentation.
-:::
+// Scheduled tasks with cron triggers
+export const cronWorker = await Worker("cron-tasks", {
+  entrypoint: "./src/cron.ts",
+  crons: [
+    "0 0 * * *",     // Run daily at midnight UTC
+    "0 */6 * * *",   // Run every 6 hours
+    "0 12 * * MON"   // Run Mondays at noon UTC
+  ]
+});
 
-## Cron Triggers
```

Comment:
This seems like a useful heading to lose. People should be able to scan the page's TOC and find the feature they're looking for?